runtime/darwinia/src/pallets/assets.rs

```diff
@@ -19,6 +19,11 @@
 // darwinia
 use crate::*;
 
+/// List of the assets existed in this runtime.
+pub enum AssetIds {
+	Kton = 1026,
```

Comment:
Is this the same for all network?

Reply:
yes, it is.

Reply:
Might move this to runtime common later.